packages/paima-sdk/paima-utils/src/config.ts

```diff
@@ -3,7 +3,7 @@ import type { VersionString } from './types/index.js';
 // https://github.com/flexdinesh/browser-or-node/blob/master/src/index.ts
 const isNode: boolean =
   typeof process !== 'undefined' && process.versions != null && process.versions.node != null;
-if (isNode) {
+if (isNode && typeof require !== 'undefined') {
```

Comment:
why is it fine to skip this? isn't the point of this to be able to run thing in node and with esm packages at the same time?

Reply:
yeah I added this piece of code a while back as a temporary hack to fix another issue I was having, and then left it in because I figured it couldn't hurt. It's maybe not a bad idea to delete it entirely

Reply:
So far this PR tweaks the imports but doesn't materially change functionality. I could trash this block if you wanted, or we can do it later (maybe in a separate PR, or in the v2 branch).

Reply:
yeah I can do this in v2, so no need to bother if you're not blocked on this